server/migrations/20190405185340-create-article.js

```diff
@@ -22,6 +22,9 @@ module.exports = {
 		images: {
 			type: Sequelize.ARRAY(Sequelize.STRING)
 		},
+		reads: {
```

Comment:
it would not be good to create new migrations when we have to make changes on the db instead of using the same old migrations then we all have to undo all migrations every time?? With this method, the rest of us wouldn't know what you changed specifically